ash/src/device.rs

```diff
@@ -33,6 +33,23 @@ impl Device {
         }
     }
 
+    pub unsafe fn from_parts(
```

Comment:
In the event that we do end up going this way, does this function strictly need to be `unsafe`? Calling any function on `Device` is already `unsafe`, and we don't implement `Drop` semantics or other safe functionality that requires `handle: vk::Device` to be valid.